src/components/InngestCommHandler.ts

```diff
@@ -756,7 +761,11 @@ export class InngestCommHandler<
 
     return {
       status: 405,
-      body: "",
+      body: JSON.stringify({
+        message: "No action found; request was likely not POST, PUT, or GET",
```

Comment:
What does this error mean? Aren't these http methods the only way to invoke the endpoint? If it's not one of these, should we log more diagnostic data?